contracts/upgradeable_contracts/BaseBridgeValidators.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity 0.4.24;
+
+import "./Ownable.sol";
+import "../libraries/SafeMath.sol";
+import "../upgradeability/EternalStorage.sol";
+
+
+contract BaseBridgeValidators is EternalStorage, Ownable {
+    using SafeMath for uint256;
+
+    address public constant F_ADDR = 0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF;
+
+    event RequiredSignaturesChanged (uint256 requiredSignatures);
+
+    function setRequiredSignatures(uint256 _requiredSignatures)
+    external
+    onlyOwner
+    {
+        require(validatorCount() >= _requiredSignatures);
+        require(_requiredSignatures != 0);
+        uintStorage[keccak256(abi.encodePacked("requiredSignatures"))] = _requiredSignatures;
+        emit RequiredSignaturesChanged(_requiredSignatures);
+    }
+
+    function getBridgeValidatorsInterfacesVersion()
```

Comment:
does it make sense to have separate versions for the rewardable validator contract and for the generic validator contract?

Reply:
The only downside I see for having separate versions is that if we change something on `BaseBridgeValidators` that will affect both validator contracts so we will have to update both versions. On the bridge contracts we have only one version that is used for all bridge modes that is defined on `BasicBridge`, so I think it's a good idea to follow that approach here too. Does it make sense?

Reply:
Ok. If it is required in the future we can address it later.